packages/react/src/metadata.tsx

```diff
@@ -315,6 +321,8 @@ export const useActionMetadata = (actionFunction: ActionFunction<any, any, any,
       action: actionName,
       includeRelatedFields: false,
     };
+  } else {
```

Comment:
This doesn't seem to me to be a 100% safe change -- are you sure that we want to throw an error here? Could there be other cases that were being silently handled by the permissive else that will now start throwing when they shouldnt?w